copier.yml

```diff
@@ -0,0 +1,83 @@
+# Configure jinja2 defaults to make syntax highlighters happy
+_templates_suffix: .jinja
+_envops:
+  block_end_string: "%}"
+  block_start_string: "{%"
+  comment_end_string: "#}"
+  comment_start_string: "{#"
+  keep_trailing_newline: true
+  variable_end_string: "}}"
+  variable_start_string: "{{"
+
+# Other Copier configurations
+_exclude:
+  - /.git
+  - /.github
+  - /.gitmodules
+  - /**.pyc
+  - /**.pyo
+  - /copier.yml
+  - /poetry.lock
+  - /pyproject.toml
+  - /tests
+  - /vendor
+
+odoo_version:
+  type: float
+  default: 13.0
+  choices:
+    - 13.0
+  help: Which Odoo version are we deploying in this branch?
+
+repo_slug:
+  type: str
+  help:
+    What's the repo slug? If you are creating https://github.com/OCA/server-tools, then
+    write "server-tools" here.
+
+repo_name:
+  type: str
+  help:
+    Tell me the project name. It's supposed to be human-readable. So, server-tools
+    project could be named like "Tools for server environment(s)"
+
+repo_description:
+  type: str
+  help: Please write a short description about what this repo is about.
```

Comment:
Can we have a rich markdown description?

Reply:
Right now multiline input is not supported in copier, at least not interactively: https://github.com/pykong/copier/issues/210

However, you can pass multiline input in `--data`.

Also, you can write any text you want as long as it's one line, which includes any markdown markup.